parser_test.go

```diff
@@ -32,9 +34,14 @@ func init() {
 	jwtTestDefaultKey = test.LoadRSAPublicKeyFromDisk("test/sample_key.pub")
 	jwtTestEC256PublicKey = test.LoadECPublicKeyFromDisk("test/ec256-public.pem")
 
+	// Load cognito public key - note there is only a public key for this key pair and should only be used for the
```

Comment:
I think it's okay to use cognito here because of this quirky behaviour we're trying to capture.

Do you have any specific concerns? @oxisto

Reply:
Just updated to remove cognito, doing a scan seems to indicate its gone now.

I actually think making it general may be more beneficial, and if anyone has questions, they can just look through the history and find this PR maybe to look through the discussion?

On the other hand, maybe if others have this issue, having an explicit place that has "cognito" in the codebase may help them debug/solve faster.